ppr-ui/src/composables/mhrInformation/useTransferOwners.ts

```diff
@@ -92,6 +92,11 @@ export const useTransferOwners = (enableAllActions: boolean = false) => {
     return getMhrTransferType.value?.transferType === ApiTransferTypes.TO_EXECUTOR_UNDER_25K_WILL
   })
 
+  /** Returns true when the passed transfer type is a 'TO_EXECUTOR_PROBATE_WILL' type **/
+  const isTransAffi = (type: ApiTransferTypes): boolean => {
```

Comment:
We are looking at individual types registration types in the table vs the selected transfer type currently in state. 
In this case, it would null on the Dashboard view.

Reply:
Got it!